go/service/teams.go

```diff
@@ -311,6 +312,8 @@ func (h *TeamsHandler) TeamAddMembersMultiRole(ctx context.Context, arg keybase1
 		return err
 	}
 
+	// TODO: should res contain a AddMembersRes for each restricted users who
+	// were not added?
 	res, err := teams.AddMembers(ctx, h.G().ExternalG(), arg.TeamID, arg.Users)
```

Comment:
protocol decision: 
Should res contain a AddMembersRes for each restricted users who were not added? Should that AddMembersRes be the default ("empty"), or should we add another field to AddMembersRes that indicates that an attempt to add a particular member failed?